lib/streamlit/util.py

```diff
@@ -212,3 +212,21 @@ def extract_key_query_params(
             for item in sublist
         ]
     )
+
+
+def extract_single_element_lists(
+    dict: Dict[str, List[Any]]
+) -> Dict[str, Union[Any, List[Any]]]:
```

Comment:
nit: might as well make this return type `Dict[str, Any]` since `Union[Any, List[Any]]` == `Any`